### ISSUE TITLE
phalcon2 Column type does not support scale parameter

### ISSUE THREAD

**Issue by hrbxb** [phalcon/cphalcon#3288]
this is the model

```php
<?php

class IhardyUserCar extends \Phalcon\Mvc\Model
{

    /**
     *
     * @var integer
     */
    public $id;

    /**
     *
     * @var integer
     */
    public $user_id;

    /**
     *
     * @var integer
     */
    public $carserial_id;

    /**
     *
     * @var string
     */
    public $flag;

    /**
     *
     * @var string
     */
    public $carsn;

    /**
     *
     * @var integer
     */
    public $madeyear;

    /**
     *
     * @var string
     */
    public $color;

    /**
     *
     * @var string
     */
    public $position;

    /**
     *
     * @var string
     */


    /**
     *
     * @var string
     */
    public $brief;

    /**
     *
     * @var string
     */


    /**
     *
     * @var string
     */
    public $provinceletter;

    /**
     *
     * @var integer
     */
    public $region_id;

    /**
     * Independent Column Mapping.
     */

    public $lng;
    public $lat;
    public function initialize() {
        $this->belongsTo("user_id",      "IhardyUsers",     "id" );
        $this->belongsTo("carserial_id", "IhardyCarserial", "id");
        $this->belongsTo("region_id",    "IhardyRegion",    "region_id");
        /*   $this->hasManyToMany(
                "id",
                "IhardyUsers",
                "user_id", "carserial_id",
                "IhardyCarserial",
                "id"
            );*/
    }

    public function columnMap()
    {
        return array(
            'id' => 'id', 
            'user_id' => 'user_id', 
            'carserial_id' => 'carserial_id', 
            'flag' => 'flag', 
            'carsn' => 'carsn', 
            'madeyear' => 'madeyear', 
            'color' => 'color', 
            'position' => 'position',
            'brief' => 'brief',
            'provinceletter' => 'provinceletter', 
            'region_id' => 'region_id',
            'lng'=>'lng',
            'lat'=>'lat',
        );
    }
}
```

**Comment by hrbxb** [comment]
phalcon1.3 is  OK

**Comment by olivier-monaco** [comment]
Sorry but your code is too long. Can you provide a minimal use case?

**Comment by hrbxb** [comment]
I use just ,IhardyUserCar::findfirst(),it creats this error

**Comment by andresgutierrez** [comment]
This was fixed in 2.0.3
